pkg/jx/cmd/create_git_api_token.go

```diff
@@ -157,132 +149,8 @@ func (o *CreateGitTokenOptions) Run() error {
 
 // lets try use the users browser to find the API token
 func (o *CreateGitTokenOptions) tryFindAPITokenFromBrowser(tokenUrl string, userAuth *auth.UserAuth) error {
```

Comment:
Why are you not removing completely this function if is dead code?

Reply:
Can remove entirely